server/src/db/models.rs

```diff
@@ -0,0 +1,36 @@
+use super::schema::{node, person};
+use uuid::Uuid;
+
+#[derive(Queryable, Identifiable)]
+#[table_name = "node"]
+pub struct Node {
```

Comment:
what is a Node? the name seems vague.

Reply:
I could have sworn I wrote a docstring for this, but I'll add one :-\

GraphQL itself doesn't force users into a graph-based view of the world, but Relay (the bridge between a React client and a GraphQL server) does - its assumption is that the objects modeled via GraphQL can be considered to be either Nodes or Edges in a graph. Following this assumption, [Relay's GraphQL Server Specification](https://relay.dev/docs/en/graphql-server-specification.html#object-identification) requires the schema to have a Node interface with a field id which is an opaque unique identifier for the node, and an endpoint `node(id: ID!)` which it can use to fetch any node.

Reply:
You can consider `Node` to be a mixin (backed by the `node` table) which adds this functionality to any object modeled as a node in the graph (in our case, Person, Group, and Transaction will all be nodes)